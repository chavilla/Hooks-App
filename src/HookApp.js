import React, { useState } from "react";

const HookApp = () => {
  const [state, setState] = useState(1);

  return (
    <>
      <h1>Hola mundo 2</h1>
      <p>{state}</p>
      <button onClick={() => setState(state + 1)}>aumenta</button>
    </>
  );
};

export default HookApp;
