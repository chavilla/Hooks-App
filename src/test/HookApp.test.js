import { shallow } from 'enzyme';
import React from 'react';
import HookApp from '../HookApp';

describe('description HookApp component', () => {
    test('should create the snapshot', () => {

        const wrapper=shallow(<HookApp />);

        expect(wrapper).toMatchSnapshot();
    })
    
})