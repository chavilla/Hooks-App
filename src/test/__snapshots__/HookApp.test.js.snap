// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`description HookApp component should create the snapshot 1`] = `
<Fragment>
  <h1>
    Hola mundo 2
  </h1>
  <p>
    1
  </p>
  <button
    onClick={[Function]}
  >
    aumenta
  </button>
</Fragment>
`;
